### PR TITLE
chore(support): fix broken support links

### PR DESCRIPTION
**Pull request by monicawheeler**
## Description
Updates Support page on Rails doc site

## Screenshots
|  Before  |  After  |
|--------|--------|
|Screen Shot 2022-08-09 at 2 26 47 PM |Screen Shot 2022-08-11 at 3 46 10 PM|

## Testing in `sage-lib`
1. Navigate to http://localhost:4000/pages/support
2. Verify the links are working

## Testing in `kajabi-products`
1. N/A

### CODE REVIEW

File: docs/app/views/pages/support.html.erb [hunk 1]
```diff
@@ -1,27 +1,22 @@
...
-[Read Our Slack Guides](https://github.com/Kajabi/UXD-Handbook/blob/main/communication/slack.md)
+We believe in being highly collaborative. Head to our [Slack Support Channel](https://slack.com/app_redirect?channel=C01A424HY8Y) and join the conversation.
 ", use_sage_type: true) %>
   </div>
   <div class="sage-col--lg-6">
     <%= md("
-## Come join the fun 🥳
-We have taken the time to establish some key touch points to support you. From Office hours, to lunch and learns there is something for everyone looking to use the system.
+## Share your feedback 📝
+We care about continuously improving. Please share your feedback with us on our [Sage Design System](https://sage.kajabi.com/) or [Sage Design System](https://github.com/Kajabi/sage-lib/) GitHub repository.
```
Comment (pixelflips) [inline docs/app/views/pages/support.html.erb:18]: Any thoughts on adjust the second link text to be something like "Sage GitHub Repository"? IMO the links look kinda odd when they're using the same copy. 

Also out of curiosity are we planning to adjust the URL? https://sage-design-system.kajabi.com is what's currently indexed. If so we might want to consider a redirect at some point.

Comment (pixelflips) [inline docs/app/views/pages/support.html.erb:18]: Any chance we could link to /pages/index to prevent a 301?

**Review by pixelflips — APPROVED** [review]
A couple of minor comments on link text and URL(s) related to a redirect, but no show-stoppers. LGTM 👍🏼

File: docs/app/views/pages/support.html.erb (hunk 1 above)
Comment (goodwinchris) [inline docs/app/views/pages/support.html.erb:18]: Is this first link to `Sage Design System` just a circular link back to the homepage?

File: docs/app/views/pages/support.html.erb [hunk 2]
```diff
@@ -1,27 +1,22 @@
...
 <% end %>
 
 <div class="sage-row">
   <div class="sage-col--lg-6">
     <%= md("
 ## We ❤️ Slack
-We believe in overcommunication. Every question is a good question, and we encourage you to take advantage of any of our slack channels to get the perspective and support of the team.
-
-[Read Our Slack Guides](https://github.com/Kajabi/UXD-Handbook/blob/main/communication/slack.md)
+We believe in being highly collaborative. Head to our [Slack Support Channel](https://slack.com/app_redirect?channel=C01A424HY8Y) and join the conversation.
```
Comment (goodwinchris) [inline docs/app/views/pages/support.html.erb:12]: Should any/all of these links be in new tabs?

File: docs/app/views/pages/support.html.erb (hunk 1 above)
Comment (monicawheeler) [reply docs/app/views/pages/support.html.erb:18]: Great callouts! 

I definitely agree the redundant text is, well redundant 😬 

Whoops, I did not mean to use sage.kajabi! I will update the URL!

Comment (monicawheeler) [reply docs/app/views/pages/support.html.erb]: Yes.

**Comment by goodwinchris** [comment]
Looks great!

**Pull request merged**